README.md

```diff
@@ -75,41 +97,64 @@ Now, configure the application in the _Settings_ tab.
 1. Configure `Allowed Callback URLs` (comma-separated)
    (The format/suffix of these urls should not be altered.)
 
-   For example, you should include `localhost` and optionally your Netlify app url(s), if you have set up Netlify:
+   You should include `localhost`. Also, you can optionally include your
+   Netlify app url(s), if you have set up Netlify.
+   BCP: What is netlify, and why would I set it up?
```

Comment:
It is a platform for hosting static websites. It takes our latest React site from git, builds it and deploys it to a CDN automatically. So actually it's not required at all - you could host the static part of the app wherever you want. For the local development case, you're just using a server on your local machine and maybe exposing it through a tunnel.